lib/src/common/model/checks.dart

```diff
@@ -45,6 +45,7 @@ class CheckRunConclusion extends EnumWithValue {
   static const neutral = CheckRunConclusion._('neutral');
   static const cancelled = CheckRunConclusion._('cancelled');
   static const timedOut = CheckRunConclusion._('timed_out');
+  static const skipped = CheckRunConclusion._('skipped');
```

Comment:
Let's add a test to cover the `skipped` case: https://github.com/SpinlockLabs/github.dart/blob/master/test/unit/checks_test.dart.